### PR TITLE
Fix: Revert Vite addition + Apps now have correct themes

### PR DESCRIPTION
**Pull request by Mycroft-Studios**
Adding vite, completely broke the phone. 
Apps also didnt seem to follow the themes correctly - text would be pure black, on dark mode.

Before:
image

After: 
image

**Pull Request Checklist**:
* [x] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [x ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change?
* [x ] Have you built and tested NPWD in-game after the relevant change?

@itschip @LiamDormon

### CODE REVIEW

File: apps/phone/src/apps/camera/components/NewPhotoButton.tsx
Comment (AvarianKnight) [inline apps/phone/src/apps/camera/components/NewPhotoButton.tsx:27]: This doesn't follow the rest of the projects use of lowerCamelCase

File: apps/phone/src/apps/contacts/components/views/ContactInfo.tsx
Comment (AvarianKnight) [inline apps/phone/src/apps/contacts/components/views/ContactInfo.tsx:58]: un-needed comment here (but is funny 🤣 )

File: apps/phone/src/apps/dialer/components/views/DialerHistory.tsx
Comment (AvarianKnight) [inline apps/phone/src/apps/dialer/components/views/DialerHistory.tsx:39]: lowerCamelCase

File: apps/phone/src/apps/match/components/views/ProfileEditor.tsx
Comment (AvarianKnight) [inline apps/phone/src/apps/match/components/views/ProfileEditor.tsx:23]: lowerCamelCase

**Review by AvarianKnight — CHANGES_REQUESTED** [review]
Only real issue I see is using `phone_name` instead of `phoneName`

**Comment by Mycroft-Studios** [comment]
> Only real issue I see is using `phone_name` instead of `phoneName`

Fixed :)

**Review by AvarianKnight — APPROVED** [review]
LGTM will wait on chips review though before merge :)

File: apps/phone/src/apps/contacts/components/List/SearchContacts.tsx
Comment (jfrader) [inline apps/phone/src/apps/contacts/components/List/SearchContacts.tsx:22]: why tailwind ... ACK REJECTED



jk LGTM

File: apps/phone/src/apps/settings/components/SettingItem.tsx
Comment (jfrader) [inline apps/phone/src/apps/settings/components/SettingItem.tsx:48]: Theme ?

File: apps/phone/src/apps/settings/components/SettingItem.tsx
Comment (jfrader) [inline apps/phone/src/apps/settings/components/SettingItem.tsx:119]: D:

File: apps/phone/src/apps/settings/components/SettingsApp.tsx
Comment (jfrader) [inline apps/phone/src/apps/settings/components/SettingsApp.tsx:166]: Why passing theme as prop instead of using theme when needed?

File: packages/ui/package.json
Comment (jfrader) [inline packages/ui/package.json:11]: nice

**Comment by jfrader** [comment]
I'm bored and miss npwd sometimes :kiss:

File: apps/phone/src/apps/settings/components/SettingsApp.tsx
Comment (Mycroft-Studios) [reply apps/phone/src/apps/settings/components/SettingsApp.tsx:166]: Except, I am using it when needed....
Anywhere i put it, it's required for the phone to work correctly with themes

File: packages/ui/package.json
Comment (Mycroft-Studios) [reply packages/ui/package.json:11]: Yes, that's what happens when you revert broken commits 😅

File: apps/phone/src/apps/contacts/components/List/SearchContacts.tsx
Comment (Mycroft-Studios) [reply apps/phone/src/apps/contacts/components/List/SearchContacts.tsx:22]: Yes, that's what happens when you revert broken commits 😅

File: apps/phone/src/apps/settings/components/SettingItem.tsx
Comment (Mycroft-Studios) [reply apps/phone/src/apps/settings/components/SettingItem.tsx:48]: theme

**Review by itschip — APPROVED** [review]

**Comment by itschip** [comment]
@Mycroft-Studios looks like lockfile is outdated from the CI. Try installing deps locally, and see if you'll get any changes locally. If not just delete, generate it, and then push the changes.

**Comment by Mycroft-Studios** [comment]
> @Mycroft-Studios looks like lockfile is outdated from the CI. Try installing deps locally, and see if you'll get any changes locally. If not just delete, generate it, and then push the changes.

will do :)

**Comment by TonybynMp4** [comment]
> > @Mycroft-Studios looks like lockfile is outdated from the CI. Try installing deps locally, and see if you'll get any changes locally. If not just delete, generate it, and then push the changes.
> 
> will do :)

do 🔫

**Comment by GlitchOo** [comment]
Should be updated in PR https://github.com/project-error/npwd/pull/1004

**Pull request merged**
